### PR TITLE
cli: refactor wallet import related commands

### LINKED ISSUE #2882: `--name` flag could not be used
`neo-go wallet import` command allows mentioning `--name` flag that should add an optional label to the resulting wallet. However, now it is broken and prints the `Enter the name of the account >` prompt despite the `--name` flag provision.

### LINKED ISSUE #2883: `Password` in CLI's configuration files
Some `neo-go` CLI commands allow fetching run parameters via a config file (`--wallet-config` flag). [Password](https://github.com/nspcc-dev/neo-go/blob/0c43b1ddca2f18fb1914a5b52bd334820e8dbf2e/pkg/config/wallet_config.go#L6) is one of such params. However, `neo-go wallet import` command [ignores](https://github.com/nspcc-dev/neo-go/blob/0c43b1ddca2f18fb1914a5b52bd334820e8dbf2e/cli/wallet/wallet.go#L623) it. It could be used instead of the `"Enter new password > "` prompt for a new account creation OR instead of the `"Enter password > "` prompt for NEP-2 strings decryption.

Useful for automation. Other commands may be improved too, did not check.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #2882, close #2883.

### CODE REVIEW

File: cli/wallet/wallet.go [hunk 1]
```diff
@@ -966,31 +977,54 @@ func ReadWalletConfig(configPath string) (*config.Wallet, error) {
 	return cfg, nil
 }
 
-func newAccountFromWIF(w io.Writer, wif string, scrypt keys.ScryptParams) (*wallet.Account, error) {
+func newAccountFromWIF(w io.Writer, wif string, scrypt keys.ScryptParams, label *string, pass *string) (*wallet.Account, error) {
+	var (
+		phrase, name string
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:982]: I'm not sure why you have both label/pass and phrase/name.

File: cli/wallet/wallet.go [hunk 2]
```diff
@@ -966,31 +977,54 @@ func ReadWalletConfig(configPath string) (*config.Wallet, error) {
...
+				return nil, fmt.Errorf("error reading password: %w", err)
+			}
 		}
 
-		return wallet.NewAccountFromEncryptedWIF(wif, pass, scrypt)
+		acc, err := wallet.NewAccountFromEncryptedWIF(wif, phrase, scrypt)
+		if err != nil {
+			return nil, err
+		}
+		acc.Label = name
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:1017]: So you won't ask for a NEP-2 name?

File: cli/wallet/wallet.go [hunk 3]
```diff
@@ -966,31 +977,54 @@ func ReadWalletConfig(configPath string) (*config.Wallet, error) {
...
+		return acc, nil
 	}
-
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:980]: :eyes:

File: cli/wallet/wallet.go [hunk 4]
```diff
@@ -966,31 +977,54 @@ func ReadWalletConfig(configPath string) (*config.Wallet, error) {
...
+			return nil, fmt.Errorf("failed to read account label: %w", err)
+		}
+	}
+	if pass == nil {
+		phrase, err = readNewPassword()
+		if err != nil {
+			return nil, fmt.Errorf("failed to read new password: %w", err)
+		}
 	}
-
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:991]: :eyes:

File: cli/wallet/wallet.go (hunk 2 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:1017]: Yes, it's done for compatibility with the old behaviour. Do we need to ask it if it's not provided via config?

File: cli/wallet/wallet.go (hunk 1 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:982]: It's done to distinguish empty label/pass from the missing one. Do we need to remove `phrase` and `name`?

File: cli/wallet/wallet.go (hunk 4 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:991]: Fixed.

File: cli/wallet/wallet.go (hunk 3 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:980]: Fixed.

File: cli/wallet/wallet.go (hunk 2 above)
Comment (roman-khimov) [reply cli/wallet/wallet.go:1017]: I think it's inconsistent, NEP-2 only differs from WIF in that it's encrypted. But new account should have a label. If we're in interactive mode it's OK to ask for a name, if we have it from a flag, it's not an issue already. And NEP-2 is not used very often for imports, to be fair, mostly people transfer keys using WIFs.

You can make it a separate patch though to be scheduled for 0.102.0.

File: cli/wallet/wallet.go (hunk 1 above)
Comment (roman-khimov) [reply cli/wallet/wallet.go:982]: You can't easily do `*pass, err = input.ReadPassword(...)`, that'd be the most convenient thing, so it's either phrase/name or `pass = &ret_from_readpwd`. OK, let's have it as is.

Comment (AnnaShaleva) [reply cli/wallet/wallet.go:982]:
```
You can't easily do *pass, err = input.ReadPassword(...)
```
Yes, that was the reason for keeping it this way.

File: cli/wallet/wallet.go (hunk 2 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:1017]: https://github.com/nspcc-dev/neo-go/pull/2889

File: cli/wallet/wallet.go (hunk 1 above)
Comment (roman-khimov) [reply cli/wallet/wallet.go:982]: Reminds me of #2671, btw. This would be easier to handle with `[]byte`.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**